.github/workflows/upload_binary.yml

```diff
@@ -29,10 +28,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up Python ${{ matrix.python-version }}
+      - name: Set up latest Python
         uses: actions/setup-python@v2
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: "3.x"
```

Comment:
My idea is to just move to latest, and only oin if we have to.

mom a big believer in move forward and rather than add tech debt by pinning by default, only do it when problems arise.

Reply:
Yeah that sounds reasonable to me, but I'll wait for other comments as you requested before changing it!

Reply:
Yes, let's just remove the pinning.